dune-project

```diff
@@ -55,6 +55,7 @@
     (>= "0.1")
     :dev))
   (odoc :with-doc)
+  crunch
```

Comment:
Thanks! Could you mark it as a `dev` dependency ? `(crunch :dev)` should be enough.